modules/misc/src/auxkernels/CoupledDirectionalMeshHeightInterpolation.C

```diff
@@ -44,8 +45,8 @@ CoupledDirectionalMeshHeightInterpolation::computeValue()
 {
   const Node & current_pos = *_current_node;
 
-  Real percentage_along_direction =
+  const Real fraction_along_direction =
       (current_pos(_direction) - _direction_min) / (_direction_max - _direction_min);
```

Comment:
Yeah, probably doesn't make much difference either way. I'm inclined to leave it as-is. I just changed this because I was reading the code trying to understand what this does, and noted that it wasn't actually a percentage.